lib/fluent/plugin/filter_masking.rb

```diff
@@ -15,11 +15,18 @@ def strToHash(str)
       # error safe method - if any error occurs the original record is returned
       def maskRecord(record)
         maskedRecord = record
-        
-        begin
+        excludedFields = []
+        @fieldsToExcludeJSONPathsArray.each do | field |
+          if record.dig(*field.split(".")) != nil
```

Comment:
save `record.dig(*field)` into a variable and then reuse it on the next line without calling `record.dig()` again